composer.json

```diff
@@ -17,6 +17,11 @@
         "psr/log": "*"
     },
     "require-dev": {
+        "phpunit/phpunit": "^4.5",
+        "squizlabs/php_codesniffer": "^2.2",
+        "phploc/phploc": "^2.0",
+        "phpmd/phpmd": "^2.2",
+        "sebastian/phpcpd": "^2.0",
```

Comment:
I’m not really a fan of adding all those tools in `require-dev`. I much prefer a Scrutinizer solution, which centralises quality assurance. Maybe try what this `.scrutinizer.yml` will yield?

```yaml
imports:
  - php

tools:
  external_code_coverage: true
```